### PR TITLE
📄 added table of content inside README

### PR DESCRIPTION
**Pull request by nitinmewar**
added table of content inside README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# Table of Content
+
+- [CodeLabz](#codelabz)
```
Comment (Sarfraz-droid) [inline README.md:3]: Try adding like this instead

1. [CodeLabz](#codelabz)
2. [Deployed on](#deployed-on)
  1. [User Guide](#user-guide)
      1. [How to Setup](#how-to-setup)
      2. [How to Use](#how-to-use)
  2. [Set-up-Firebase](#set-up-firebase)
      1. [How to Deploy Firebase Cloud Functions](#how-to-deploy-firebase-cloud-functions)

File: README.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+      - [How to Setup](#how-to-setup)
+      - [How to Use](#how-to-use)
+  * [Set-up-Firebase](#set-up-firebase)
+      - [How to Deploy Firebase Cloud Functions](#how-to-deploy-firebase-cloud-functions)
+- [Firebase Emulator Flow](#firebase-emulator-flow)
+- [Install, Configure and Integrate Firebase Emulator Suite](#install--configure-and-integrate-firebase-emulator-suite)
+  * [**How to install**](#--how-to-install--)
+      - [Before installing the Emulator Suite you will need:-](#before-installing-the-emulator-suite-you-will-need--)
+      - [To install the Emulator Suite:](#to-install-the-emulator-suite-)
+  * [**How to Configure**](#--how-to-configure--)
```
Comment (Sarfraz-droid) [inline README.md:15]:
```
//Not working
  * [**How to install**](#--how-to-install--)
      - [Before installing the Emulator Suite you will need:-](#before-installing-the-emulator-suite-you-will-need--)
      - [To install the Emulator Suite:](#to-install-the-emulator-suite-)
  * [**How to Configure**](#--how-to-configure--)
      - [Configure Emulator Suite:-](#configure-emulator-suite--)
      - [Port Configuration - These are the Default Ports Provided you can change them while initializing(if needed)](#port-configuration---these-are-the-default-ports-provided-you-can-change-them-while-initializing-if-needed-)
      - [To Start the Emulators:-](#to-start-the-emulators--)
      - [Import and Export:-](#import-and-export--)
```

**Review by Sarfraz-droid — CHANGES_REQUESTED** [review]

**Comment by Sarfraz-droid** [comment]
@nitinmewar have a look on the requested changes above

**Review by Sarfraz-droid — APPROVED** [review]

**Comment by Sarfraz-droid** [comment]
LGTM

**Review by Maahi10001 — APPROVED** [review]

**Pull request merged**